keep/api/routes/incidents.py

```diff
@@ -0,0 +1,392 @@
+import logging
+import os
+import pathlib
+import sys
+from typing import List
+
+from fastapi import APIRouter, Depends, HTTPException, Response
+from pusher import Pusher
+from pydantic.types import UUID
+
+from keep.api.core.db import (
+    add_alerts_to_incident_by_incident_id,
+    assign_alert_to_incident,
+    create_incident_from_dto,
+    delete_incident_by_id,
+    get_incident_alerts_by_incident_id,
+    get_incident_by_id,
+    get_last_alerts,
+    get_last_incidents,
+    remove_alerts_to_incident_by_incident_id,
+    update_incident_from_dto_by_id, confirm_predicted_incident_by_id, create_incident_from_dict,
+)
+from keep.api.core.dependencies import (
+    AuthenticatedEntity,
+    AuthVerifier,
+    get_pusher_client,
+)
+from keep.api.models.alert import AlertDto, IncidentDto, IncidentDtoIn
+from keep.api.utils.enrichment_helpers import convert_db_alerts_to_dto_alerts
+from keep.api.utils.pagination import IncidentsPaginatedResultsDto
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+ee_enabled = os.environ.get("EE_ENABLED", "false") == "true"
+if ee_enabled or 1:
+    path_with_ee = (
+        str(pathlib.Path(__file__).parent.resolve()) + "/../../../ee/experimental"
```

Comment:
99% will break soon

Reply:
I'm not sure how to guarantee import without such a relative import. If it will be an absolute import, it will depend on the folder we launch the app from. I could add a unit test